spec/routing/external_link_routing_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe ExternalLinkController do
+  describe 'routing' do
+    context 'when locale is en' do
+      let(:id) { SecureRandom.uuid }
+
+      it 'returns route to #show' do
+        expect(get("/team/#{id}/blog")).to route_to(
+          controller: 'external_link', action: 'show',
```

Comment:
Can we have each key in separate line. It will be more readable.

Reply:
We have settings that allow us to have only 5 lines in a single test case.
At the beginning I had:
```
expect(get("/team/#{id}/blog")).to route_to(
  controller: 'external_link', 
  action: 'show',
  person_id: id, 
  link_name: 'blog', 
  locale: 'en'
)
```
It didn't pass the Rubocop tests.
I changed the formatting to:
```
expect(get("/team/#{id}/blog")).to route_to(controller: 'external_link', 
  					    action: 'show',
                                            person_id: id, 
                                            link_name: 'blog', 
                                            locale: 'en')
```
But Alek didn't like it and he decided to change the formatting to what it is now.

Reply:
I was called to answer. Overall, @womanonrails is right. This is a test file, so the number of lines shouldn't matter that much. It's more important that the test is readable and understandable than that it meets the 5-line rule per method. Let's try to write concise methods, but not at the expense of readability.

BTW If I remember correctly, at the beginning you had everything in one line. :wink: